contracts/ERC5018ForBlob.sol

```diff
@@ -123,18 +129,19 @@ contract ERC5018ForBlob is IERC5018ForBlob, Ownable {
         require(msg.value >= cost * length, "insufficient balance");
 
         for (uint8 i = 0; i < length; i++) {
-            require(sizes[i] <= 4096 * 31, "invalid blob length");
+            require(sizes[i] <= DECODE_BLOB_SIZE, "invalid chunk length");
             _preparePut(key, chunkIds[i]);
 
             bytes32 chunkKey = keccak256(abi.encode(msg.sender, block.timestamp, chunkIds[i], i));
-            storageContract.putBlob{value : cost}(chunkKey, i, sizes[i]);
+            storageContract.putBlob{value : cost}(chunkKey, i, BLOB_SIZE);
```

Comment:
May I know the reason why the contract wants to manage the size by itself?

Reply:
The original file content after blob decode can now be retrieved, rather than the uploaded blob content, so the size of the original file is required.